command/job_start.go

```diff
@@ -162,3 +190,24 @@ func (c *JobStartCommand) Run(args []string) int {
 	mon := newMonitor(c.Ui, client, length)
 	return mon.monitor(resp.EvalID)
 }
+
+func parseFromSubmission(sub *api.JobSubmission) (*api.Job, error) {
+	var job *api.Job
+	var err error
+
+	switch sub.Format {
+	case "hcl2":
+		job, err = jobspec2.Parse("", strings.NewReader(sub.Source))
+		if err != nil {
+			return nil, fmt.Errorf("command: unable to parce job submission: %w", err)
```

Comment:
The error we return to the user here isn't going to explain why we're trying to parse the job submission in the first place. What if it was:

```suggestion
			return nil, fmt.Errorf("Unable to parse job submission to re-enable scaling policies: %w", err)
```